[CI] FiltersAggregatorTests testMatchAll failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/xzbwxjrdr32kq/tests/:server:test/org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests/testMatchAll

**Reproduction line:**
`./gradlew ':server:test' --tests "org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.testMatchAll" -Dtests.seed=CDDFCC0AE26A5A12 -Dtests.locale=ar-LB -Dtests.timezone=Etc/GMT-2 -Druntime.java=8`

**Applicable branches:**
7.14

**Reproduces locally?:**
Yes

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests&tests.test=testMatchAll

**Failure excerpt:**
```
java.lang.AssertionError: Expected a map containing
                       test: a map containing
             segments_counted: <2>
           segments_collected: <0>
segments_with_doc_count_field: <0>
   segments_with_deleted_docs: <0>
                      filters: a list containing
                              0: a map containing
                            query: "*:*"
                  specialized_for: "match_all"
            results_from_metadata: expected <1> but was <2>

  at __randomizedtesting.SeedInfo.seed([CDDFCC0AE26A5A12:A19E02592C99E2F8]:0)
  at io.github.nik9000.mapmatcher.MapMatcher.assertMap(MapMatcher.java:78)
  at io.github.nik9000.mapmatcher.MapMatcher.assertMap(MapMatcher.java:61)
  at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.lambda$testMatchAll$16(FiltersAggregatorTests.java:471)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.debugTestCase(AggregatorTestCase.java:640)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.lambda$debugTestCase$9(AggregatorTestCase.java:595)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.withIndex(AggregatorTestCase.java:559)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.debugTestCase(AggregatorTestCase.java:595)
  at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.testMatchAll(FiltersAggregatorTests.java:462)
  at sun.reflect.NativeMethodAccessorImpl.invoke0(NativeMethodAccessorImpl.java:-2)
  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:498)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1750)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:938)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:974)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:988)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:817)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:468)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:947)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:832)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:883)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:894)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
  at java.lang.Thread.run(Thread.java:748)

```

--- a/server/src/test/java/org/elasticsearch/search/aggregations/bucket/filter/FiltersAggregatorTests.java
+++ b/server/src/test/java/org/elasticsearch/search/aggregations/bucket/filter/FiltersAggregatorTests.java
@@ -460,7 +460,7 @@ public class FiltersAggregatorTests extends AggregatorTestCase {
                                 matchesList().item(
                                     matchesMap().entry("query", "*:*")
                                         .entry("specialized_for", "match_all")
-                                        .entry("results_from_metadata", 1)
+                                        .entry("results_from_metadata", greaterThanOrEqualTo(1))
                                 )
                             )
                     )

